PA1Q1

```diff
@@ -0,0 +1,94 @@
+# COMP3410 Program Template
+# Author: Nathaniel Warren III
+# Assignment: PA[1]
+# Date: 2/11/2015
+# Turn in one .asm file per assignment component
+# Remember to submit it as a pull request to the GitHub repo for the assignment
+
+# fibonacci.asm
+# Input: None
+# Output: Print to terminal N Fibonacci numbers
+
+##########################################################
+# .data segment. Constant and variable definitions go here.
+##########################################################
+
+	.data
+
+fibs:	.word   0 : 9         # create an array variable named "fibs" of 9 word-length elements (4 bytes each)
+size:	.word  9              # create a single integer variable named "size" that indicates the length of the array
+text: .ascii "Enter a number between 1-9\n" 
+###############################################
+# .text segment. Assembly instructions go here.
+###############################################
+	.text
+	      la   $s0, fibs        # load address of array into $s0
+	      #la   $s5, size        # load address of size variable into $s5
+	      #lw   $s5, 0($s5)      # load array size from its address in the register
+
+	      li $v0, 4           #tells system to prepare for pringing a sting to terminal
+	    la $a0, text         #placing text specified above in a0 - this is printed to screen
+    	syscall
+	
+	    li $v0,5 #accepts user input goes to screen
+	    syscall
+	move $s5, $v0
```

Comment:
-5 Accepts input greater than 9.  This allows storing numbers beyond the bounds of the array.